main.py

```diff
@@ -79,6 +79,33 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 
     await ctx.send(content = "Done", delete_after = 0.1)
 
+@slash.slash(name="IsEveryoneHere", description="Checks if all working 'Full Time' members are in voice channel", guild_ids=guild_ids)
+async def IsEveryoneHere(ctx):
+
+    today = datetime.today().date()
+    today = datetime(today.year, today.month, today.day)
+
+    guild = client.guilds[0]
+    meeting_channel = client.get_channel(int(os.getenv("MeetingChannel_id")))
+    fulltime_role = discord.utils.get(guild.roles, name = "Full Time")
+    
+    fulltime_members = list(filter(lambda member: fulltime_role in member.roles, guild.members))
+    fulltime_members_in_voicechannel = list(filter(lambda member : fulltime_role in member.roles, meeting_channel.members))
+    not_here = list(set(fulltime_members) - set(fulltime_members_in_voicechannel))
+
+    approved_leaves = list(filter(lambda member : (leave_interface.IsMemberOnLeave(member.id, today))[0], not_here))
+    approved_leaves_names = [member.display_name for member in approved_leaves]
+    approved_leaves_reasons = [leave_interface.IsMemberOnLeave(member.id, today)[1] for member in approved_leaves]
+    approved_leaves_dict = dict(zip(approved_leaves_names, approved_leaves_reasons))
```

Comment:
this can be done in a separate function `GetApprovedLeavesForDay(member_list, day)`

Reply:
I initially had it as a separate function but couldn't find an appropriate place for it without cyclic imports. After speaking with Anthony about it we decided to keep it in main as is.